packages/@react-native-windows/cli/src/index.ts

```diff
@@ -6,6 +6,19 @@
 
 import * as fs from 'fs';
 import * as path from 'path';
+import * as findUp from 'find-up';
+
+import * as appInsights from 'applicationinsights';
+
+appInsights.setup('795006ca-cf54-40ee-8bc6-03deb91401c3');
+const telClient = appInsights.defaultClient;
+if (!telClient.commonProperties.sessionId) {
+  telClient.commonProperties['sessionId'] = (new Date()).getTime().toString(36) + Math.random().toString(36).slice(2);
+}
+if (process.env.RNW_CLI_TEST) {
```

Comment:
OOC what is this used for?

Reply:
this gives us a way to separate telemetry used for testing purposes (like I'm doing when testing these changes) so that it doesn't pollute production telemetry (i.e. filter by isTest != 'true')